### PR TITLE
Fixed: Unable to deploy and test the kubernetes documentation site locally in windows 

### PR DESCRIPTION
**Pull request by PranitRout07**
**ISSUE DETAILS**
Windows commands are not mentioned properly in the README.md file to deploy and test the site locally. Even i found out that previously many users also have issue with windows and many contributors are continuously asking the maintainers the same question that how to deploy the site and test in windows locally.

refer #43401 
I have changed the approach that i used in the above pr.

**WHAT I HAVE CHANGED?**

All the commands should be written in command prompt while deploying the site locally.
Added the following commands:

rmdir /s /q i18n  ->  _This command will first remove the i18n directory_

for /f %A in ('cd') do mklink /J "%A\i18n" "%A\data\i18n"   -> _This command will make symbolic link in windows._

### CODE REVIEW

**Review by aj11anuj — COMMENTED** [review]
I don't think, there are much people in today's era that still uses command prompt CLI of Windows. If it will work for powershell, or can be made to work for powershell as well then it's fine, otherwise it may not be feasible.

**Comment by PranitRout07** [comment]
Hey @aj11anuj  , i have updated the commands so that it will work on powershell.

File: README.md [hunk 1]
```diff
@@ -62,11 +62,15 @@ To install dependencies, deploy and test the site locally, run:
 
 - For macOS and Linux
   ```bash
+
```
[the remark this answers is not in the document]
Comment (PranitRout07) [reply README.md:65]: There is no need for empty newline. Thanks for the review:)

File: README.md [hunk 2]
```diff
@@ -61,12 +61,16 @@ Make sure to install the Hugo extended version specified by the `HUGO_VERSION` e
...
+  
   ```bash
   npm ci
   make serve
   ```
 - For Windows (PowerShell)
+  
   ```powershell
+  Remove-Item -Path "i18n" -Recurse -Force
+  $CD = Get-Location; New-Item -ItemType Junction -Path "$CD\i18n" -Value "$CD\data\i18n"
```
Comment (sftim) [inline README.md:73]: We'd need to warn people not to commit the change they make here. Is there any alternative that doesn't involve running `Remove-Item`?

Comment (sftim) [reply README.md:73]: Maybe a PowerShell one liner to convert the symlinks into something Windows native?

Comment (PranitRout07) [reply README.md:73]: I'm currently looking into converting the symlinks into something more native to Windows using PowerShell. I appreciate your patience as I work on finding the solution. I'll make sure to update you as soon as I have it sorted out

Comment (PranitRout07) [reply README.md:73]: Hey @sftim , i have just tested deploying the site locally , found out that we can now easily access the site locally without any issue on Windows . I think this problem is resolved as we upgraded to node version 20.11.0 and hugo version 0.121.2 . When i made this PR at that time we are using other version of node and hugo .So in my opinion it is good to close this PR .

Comment (sftim) [reply README.md:73]: Feel free to close your PR!

**Pull request closed without merging**